test/unit/specs/store/helpers/helpers.spec.js

```diff
@@ -1,5 +1,9 @@
 import { expect } from 'chai'
-import helpers, {CODE_REGEX} from '../../../../../src/store/helpers'
```

Comment:
u can use '@/src/store/helpers' to avoid having to number of '../' , this can also make refactoring / moving stuff easer

Reply:
I tried but it doesn't work, see https://jenkins.dev.molgenis.org/blue/organizations/jenkins/molgenis%2Fmolgenis-app-biobank-explorer/detail/PR-68/18/pipeline

Reply:
it seems to work when i try;
https://jenkins.dev.molgenis.org/blue/organizations/jenkins/molgenis%2Fmolgenis-app-biobank-explorer/detail/PR-70/1/pipeline

Reply:
Aha! Updating.

Reply:
See #72